doc/engineering-reference/src/simulation-models-encyclopedic-reference-005/occupant-thermal-comfort.tex

```diff
@@ -336,163 +364,224 @@ \subsubsection{Pierce Two-Node Model Nomenclature List}\label{pierce-two-node-mo
+Q\(_{dry}\) or DryHeatLoss is calculated in the EnergyPlus code using the following equation:
 
 DryHeatLoss = CloBodyRat*(Hc*(CloSurfTemp - AirTemp) + Hr*(CloSurfTemp - RadTemp))
 
-In Pierce model, the convective heat transfer coefficient, h\(_{c}\), varies with the air velocity around body and metabolic rate. The model uses the maximum value of following equations.
+In Pierce model, the convective heat transfer coefficient, h\(_{c}\), varies with the air velocity around body and metabolic rate. The model uses the maximum value of following equations:
 
-\({h_c} = 8.6 \times {v^{_{0.53}}}\) ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ W/m\(^{2}\)°C
+\begin{equation}
+{h_c} = 8.6 \times {v^{_{0.53}}}
+\end{equation}
 
-\({h_c} = 5.66{(M/58.2 - 0.85)^{0.39}}\) ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ W/m\(^{2}\)°C
+\begin{equation}
+{h_c} = 5.66{(M/58.2 - 0.85)^{0.39}}
+\end{equation}
+
+In the EnergyPlus code, the following equations are used to calculate h\(_{c}\) or Hc:
 
-Hc = 8.6*AirVel**0.53
+Hc = 8.6*AirVel**0.53 where AirVel is reset to 0.137 if it drops below this value
 
-HcAct = 5.66*(ActMet - 0.85)**0.39
+HcAct = 5.66*(ActMet - 0.85)**0.39 when ActMet is greater than 0.9
+
+When ActMet is greater than 0.9, EnergyPlus uses the larger value of Hc and HcAct for h\(_{c}\).
 
 Also, in the model, the radiant heat transfer coefficient, h\(_{r}\), is defined by following equation (Doherty and Arens 1988):
 
-\({h_r} = 4. \times {f_{eff}}\varepsilon \sigma {(({T_{cl}} + {T_r})/2 + 273.15)^3}\) ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ W/m\(^{2}\)°C
+\begin{equation}
+{h_r} = 4. \times {f_{eff}}\varepsilon \sigma {(({T_{cl}} + {T_r})/2 + 273.15)^3}
+\end{equation}
+
+In the EnergyPlus code, h\(_{r}\) or Hr is calculated using the following equation:
 
 Hr = 4.*RadSurfEff*StefanBoltz*((CloSurfTemp + RadTemp)/2. + TAbsConv)**3
 
-In the Pierce model, T\(_{cl}\) is estimated by each iteration using following equation:
+In the Pierce model, T\(_{cl}\) in �C is estimated by each iteration using following equation:
 
-\({T_{cl}} = ({T_{sk}}/{I_{cl}} + {f_{cl}}({h_c}{T_a} - {h_r}{T_r}))/(1/{I_{cl}} + {f_{cl}}({h_c} + {h_r}))\) ~~~~~~~~~~~~~ °C
+\begin{equation}
+{T_{cl}} = ({T_{sk}}/{I_{cl}} + {f_{cl}}({h_c}{T_a} + {h_r}{T_r}))/(1/{I_{cl}} + {f_{cl}}({h_c} + {h_r}))
+\end{equation}
 
-CloSurfTemp = (CloCond*SkinTemp + CloBodyRat*(Hc*AirTemp \&
+Or as shown in the EnergyPlus code:
 
-\begin{itemize}
-\tightlist
-\item
-  Hr*RadTemp))/(CloCond + CloBodyRat*(Hc + Hr))
-\end{itemize}
+CloSurfTemp = (CloCond*SkinTemp + CloBodyRat*(Hc*AirTemp+Hr*RadTemp))
 
-Total evaporative heat loss from the skin, E\(_{sk}\), includes evaporation of water produced by regulatory sweating, E\(_{rsw}\), and evaporation of water vapor that diffuses through the skin surface, E\(_{diff}\).
+~~~~~~~~~~~~~~~~~~~~~~~/(CloCond + CloBodyRat*(Hc+Hr))
```

Comment:
Is this supposed to be attached to the equation two lines up?  If so, can you clean up the connection?  Or just make it a real equation and not a source code representation?

Reply:
@Myoldmopar Well, here's what I can say about this.  When I delete the blank line between these two lines, it merges them back together like there is no paragraph marker between them and it messes up the spacing upon building the Eng Ref on my machine.  Same thing with PMVET, PMVSET, and DISC equations elsewhere in this section.  So, I left them...or is there another way to do this?

Reply:
It's fine.  I mean, really, the right thing to do, is to put them in actual equation format, instead of just pasted in code equations.  Then LaTeX can handle this formatting stuff and we don't have to keep things lined up just right.  But I get the original intent of this section and other parts of the code were to show the actual equations from the source, so just leave them.

Reply:
@Myoldmopar Yah, I think for consistency and differentiation I was using equation format for actual mathematical equations and then leaving code as text.  At least that was my intent.  Trying to get the issue that @michael-okeefe noted sorted out.  Some of these codes are a bit temperamental...but it is pretty cool that we can get all of the documentation in version control.